templates/source/sourceTemplate.md

```diff
@@ -0,0 +1,35 @@
+# Event Source for **\_\_\_\_**
+
+This event target receives .......
+
+## Prerequisites
+
+- unordered list out prerequisites
```

Comment:
Out -> Of (or did I get that wrong?)

Reply:
sorry yes should be of. I made edits to that and didn't fix that